modules/wkt/test/wkb/encode-wkb.spec.ts

```diff
@@ -0,0 +1,143 @@
+import test from 'tape-promise/tape';
+import {fetchFile} from '@loaders.gl/core';
+import encodeWKB from '../../src/lib/encode-wkb';
+import hexStringToArrayBuffer from './hex-string-to-array-buffer';
+import {Geometry, BinaryGeometry} from '@loaders.gl/schema';
+
+const WKB_2D_TEST_CASES = '@loaders.gl/wkt/test/data/wkb-testdata2d.json';
+const WKB_2D_NAN_TEST_CASES = '@loaders.gl/wkt/test/data/wkb-testdata2d-nan.json';
+const WKB_Z_TEST_CASES = '@loaders.gl/wkt/test/data/wkb-testdataZ.json';
+const WKB_Z_NAN_TEST_CASES = '@loaders.gl/wkt/test/data/wkb-testdataZ-nan.json';
+
+interface TestCase {
+  geometry: string;
+  wkt: string;
+  wkb: string;
+  ewkb: string;
```

Comment:
Do we use all of this? Comments to help navigate the alphabet soup?

Reply:
I was planning to put this in a `utils.ts` to share with `parse-wkb.spec.ts`